test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -784,7 +795,7 @@ proc ::AlgorandGoal::AddParticipationKey { ADDRESS FIRST_ROUND LAST_ROUND TEST_P
 # Register online participation with a given account
 proc ::AlgorandGoal::TakeAccountOnline { ADDRESS FIRST_ROUND LAST_ROUND TEST_PRIMARY_NODE_DIR } {
    if { [ catch {
-       spawn goal account changeonlinestatus --address $ADDRESS --firstvalid $FIRST_ROUND --lastvalid $LAST_ROUND -d datadir
+       spawn goal account changeonlinestatus --address $ADDRESS --firstvalid $FIRST_ROUND --lastvalid $LAST_ROUND -d datadir --online
```

Comment:
this change is supposed to be a no-op. i.e. the default for `--online` is true. If this is not the case, then we have a bug, which is unrelated to this PR.

Reply:
`--online` is required by the `goal account changeonlinestatus` command.    Without it,  the command fails.  I will create a ticket to fix the command and update this test case to remove the `--online` option.

Reply:
The PR looks good to me, but I wonder how this test did not detect this issue before.
It's quite concerning that we have a test that is supposed to test exactly one thing - and the test doesn't alert us when that thing is broken.

Reply:
We found this issue by running the integration tests as part of the Jenkins/Mule pipeline testing.  The issue was causing the test test/e2e-go/cli/goal/expect/listExpiredParticipationKeyTest.exp to fail.  What is puzzling is why this error does not surface as part of the Travis build and testing.  I created this ticket to resolve: https://github.com/algorand/go-algorand-internal/issues/542